### ISSUE TITLE
No multiple commands on pane

### ISSUE THREAD

**Issue by aspeteRakete** [tmuxinator/tmuxinator#329]
Several Commands in panes don't get executed at all:

```
name: pane bug
root: ~/
attach: false
post: tmux -CC attach
windows:
  - logs:
    layout: main-vertical
    panes:
      - nginx:
        - "ssh root@example.com"
        - "cd /var/log/nginx/"
        - "tail -f error.log"
  - htop:
    - ssh root@example.com
    - htop
```

```
> tmuxinator debug bug
```

gives:

```
#!/bin/bash

# Clear rbenv variables before starting tmux
unset RBENV_VERSION
unset RBENV_DIR

tmux start-server\; has-session -t debug-edge-update\ server 2>/dev/null

if [ "$?" -eq 1 ]; then
  cd /Users/felix

  # Run pre command.


  # Create the session and the first window. Manually switch to root
  # directory if required to support tmux < 1.9
  TMUX= tmux new-session -d -s debug-edge-update\ server -n logs


  # Create other windows.
  tmux new-window  -t debug-edge-update\ server:1 -n htop


  # Window "logs"

  # Window "htop"
  tmux send-keys -t debug-edge-update\ server:1 ssh\ root@example.com C-m
  tmux send-keys -t debug-edge-update\ server:1 htop C-m

  tmux select-window -t 0
fi


tmux -CC attach
```

**Comment by neowulf33** [comment]
+1

**Comment by dav-m85** [comment]
+1

**Comment by neowulf33** [comment]
Ok, mine was a rookie mistake.

I had to look in `$HOME/.tmux.conf` and disable the following line:

```
#set -g base-index 1
```

**Comment by fabianschwarzfritz** [comment]
Hi, 
there seems to be a wrong **tmuxinator configuration**. A correct configuration works for me without any issues: 
```
name: pane bug
root: ~/
attach: false
post: tmux -CC attach
windows:
  - logs:
      layout: main-vertical
      panes:
        - nginx:
          - "ssh root@example.com"
          - "cd /var/log/nginx/"
          - "tail -f error.log"
  - htop:
    - ssh root@example.com
    - htop
```
*Notice the indentation under ```-logs``` . The debug output gives me the following result, which looks good to me: 

```
#!/bin/bash

# Clear rbenv variables before starting tmux
unset RBENV_VERSION
unset RBENV_DIR

tmux start-server;

  cd /Users/d056995

  # Run pre command.


  # Create the session and the first window. Manually switch to root
  # directory if required to support tmux < 1.9
  TMUX= tmux new-session -d -s pane\ bug -n logs
  tmux send-keys -t pane\ bug:0 cd\ /Users/d056995 C-m


  # Create other windows.
  tmux new-window  -t pane\ bug:1 -n htop


  # Window "logs"
  tmux send-keys -t pane\ bug:0.0 ssh\ root@example.com C-m
  tmux send-keys -t pane\ bug:0.0 cd\ /var/log/nginx/ C-m
  tmux send-keys -t pane\ bug:0.0 tail\ -f\ error.log C-m

  tmux select-layout -t pane\ bug:0 tiled

  tmux select-layout -t pane\ bug:0 main-vertical
  tmux select-pane -t pane\ bug:0.0

  # Window "htop"
  tmux send-keys -t pane\ bug:1 ssh\ root@example.com C-m
  tmux send-keys -t pane\ bug:1 htop C-m

  tmux select-window -t 0
  tmux select-pane -t 0


tmux -CC attach
```

All commands in the *logs* pane are executed successfully. @aspeteRakete does this still occur using a correct indentation in the tmuxinator configuration?

@adamstrickland  and other tmuxinator experts: Is there currently any validation of the tmuxinator configuration before it is executed?

Cheers and bests,
Fabian
🍺 😄 🏄 ☀️

**Comment by adamstrickland** [comment]
As far as I know, there is not.  Part of the difficulty with a problem like this is that both structures are valid YAML, so we couldn't rely on a simple YAML validation check (which tmuxinator _does_ do).  We'd need something along the lines of a schema validation; perhaps [JSON Schema](http://json-schema.org/) would be good here?

**Comment by ethagnawl** [comment]
Closed via #630. (Feel free to reopen if it makes sense.)
